zkmarek/video/slides/wallet.py

```diff
@@ -53,3 +62,19 @@ def animate_public_key(self, scene):
         self.public_key_equation = new_public_key_equation
         scene.wait()
         scene.play(FadeOut(self.public_key, self.public_key_equation))
+
+    def animate_scalar_multiplication(self, scene):
+        scene.play(FadeIn(self.chart))
+
+        generator1 = ECAffine(36, 28, self.curve)
+        subgroup1 = Subgroup.from_generator(generator1)
+        animation1 = AnimateSubgroups(self.chart, runtime_per_step=0.6)
+        animation1.animate_generator(scene, generator1, split_animation=True)
+
+        scene.wait()
+        animation1.animate_subgroup_mid(scene, subgroup1, generator1, 2, 18)
+
+        scene.wait()
+        gen_dot = self.chart.find_dot_by_affine(ECAffine(39, 9, self.curve))
```

Comment:
Misleading name, gen_dot sounds like generator, but it is in fact public key

Reply:
renamed to `public_key_point`